skill-tree/docker-compose.yml

```diff
@@ -0,0 +1,17 @@
+version: '3.1'
+
+services:
+  db:
+    image: mysql
+    environment:
+      MYSQL_ROOT_PASSWORD: 'testpassword'
+    ports:
+      - "3306:3306"
+    expose:
+      - "3306"
+    restart: always
```

Comment:
Lets not restart it by default always

Reply:
I tried understanding this, didn't get it, could you explain it @prakashchoudhary07

Reply:
restart: always, would restart the container when ever docker is started, so we should only start when needed